src/here-xyz.ts

```diff
@@ -1093,19 +976,34 @@ async function showSpace(id: string, options: any) {
 
 program
     .command("delete <id>")
-    .description("delete the XYZ space with the given id")
+    .description("delete the XYZ space or project with the given id")
     .option("--force", "skip the confirmation prompt")
     .option("--token <token>", "a external token to delete another user's space")
     .action(async (geospaceId, options) => {
         //console.log("geospaceId:"+"/geospace/"+geospaceId);
-        
+            deleteSpace(geospaceId, options)
+                .catch((error) => {
+                    handleError(error, true);
+                })
+    });
 
-        deleteSpace(geospaceId, options)
-            .catch((error) => {
-                handleError(error, true);
-            })
 
-    });
+async function deleteProject  (id : any, options: any) {
```

Comment:
do we need this method here?

Reply:
Have removed references of projects (show/delete) from here-xyz. These were kept initially in here-xyz but later got moved to here-studio.ts Forgot to update the reference here. Thanks for pointing